CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+This repository contains multiple JavaScript packages, each with multiple changelogs. Federation 1 is managed on the [`version-0.x` branch](https://github.com/apollographql/federation/tree/version-0.x), while Federation 2 is managed on the [`main` branch](https://github.com/apollographql/federation/tree/main). Please consult the table below to find the changelog you are looking for:
+
+|npm package|version|changelog|
+|--|--|--|
+|[`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway/v/latest-1)|v1|[link](https://github.com/apollographql/federation/blob/version-0.x/gateway-js/CHANGELOG.md)|
```

Comment:
**suggestion** Maybe having a `v1` and `v2` column would make this table less daunting, since almost all of the packages have v1 & v2 links

Reply:
great idea thank you